lib/ui/workspace_registeration_screen/create_workspace/workspace_steps/workspace_step2/create_workspace2_viewmodel.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/cupertino.dart';
+import 'package:stacked/stacked.dart';
+
+class CreateWorkspaceViewModel2 extends BaseViewModel {
+  final ScrollController controllerOne = ScrollController();
```

Comment:
This controller introduce the `Cupertino` import, we should not have any of those imports in the viewmodel, i.e `material `or `cupertino`, if you need scroll container a Hook widget is perfect for this.